ds-rpackaging/setup.md

```diff
@@ -27,10 +30,16 @@ $ library(roxygen2)
 ~~~
 {: .language-r}
 
-Please contact us in advance if you are experiencing problems.
```

Comment:
I would leave this here, because now its below the preparation and prior knowledge part and it's a little bit out of place!